sculture-android/app/src/main/java/tr/edu/boun/cmpe/sculture/activity/TagActivity.java

```diff
@@ -0,0 +1,211 @@
+                    tagDescription.setText(previous);
+                    save.setVisibility(View.GONE);
+                } else {
+                    previous = tagDescription.getText().toString();
+                    tagDescription.setEnabled(true);
+                    editTagDesc.setText("Cancel");
+                    save.setVisibility(View.VISIBLE);
+                }
+                is_on_edit = !is_on_edit;
+            }
+        });
+
+    }
+
+    private void setRecyclerListeners() {
+        story_list_recycler.addOnScrollListener(new RecyclerView.OnScrollListener() {
+            @Override
+            public void onScrollStateChanged(RecyclerView recyclerView, int newState) {
+                super.onScrollStateChanged(recyclerView, newState);
+                load_story();
+            }
+        });
+    }
+
+    private void load_story() {
+        int totalItemCount = mLayoutManager.getItemCount();
+        int lastVisibleIndex = mLayoutManager.findLastVisibleItemPosition();
+
+        boolean loadMore = lastVisibleIndex == totalItemCount - 1;
+
+        if (is_reach_end)
+            Log.i("HERE", "HERE");
+        if ((loadMore && !is_looding_more && !is_reach_end)) {
+            is_looding_more = true;
+            JSONObject requestBody = new JSONObject();
+            try {
+                requestBody.put(FIELD_TITLE, BUNDLE_TAG_TITLE);
+                requestBody.put(FIELD_SIZE, 10);
+            } catch (JSONException e) {
+                e.printStackTrace();
+            }
+            addRequest(API_USER_STORIES, requestBody, new Response.Listener<JSONObject>() {
+                @Override
+                public void onResponse(JSONObject response) {
+                    SearchResponse searchResponse = new SearchResponse(response);
+
+                    for(BaseStoryResponse story : searchResponse.result)
+                        mStoryListViewAdapter.addElement(story);
+
+                    if(searchResponse.result.size() == 0)
+                        is_reach_end = true;
+
+                    is_looding_more = false;
+                }
+            }, new Response.ErrorListener() {
+                @Override
+                public void onErrorResponse(VolleyError error) {
+
+                }
+            }, null);
+        }
+    }
+
+    private void getTag(String title) {
+        final JSONObject requestObject = new JSONObject();
+        try {
+            requestObject.put(FIELD_TITLE, title);
+        } catch (JSONException e) {
+            e.printStackTrace();
+        }
+
+        addRequest(API_TAG_GET, requestObject,
+                new Response.Listener<JSONObject>() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        TagResponse tag = new TagResponse(response);
+                        tagTitle.setText(tag.tag_title);
+                        tagDescription.setText(tag.description);
```

Comment:
There is also last_edit_date and last_editor_id information which should be used.

Last editor name is not retrieved by this API. I will open an issue about it.